codebase/general/src.bin/dat/datdump/doc/datdump.doc.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<binary>
```

Comment:
```suggestion
<?ignore
(C) Copyright 2021 SuperDARN Canada, University of Saskatchewan
Author: Marina Schmidt 
This file is part of the Radar Software Toolkit (RST).
RST is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.
This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.
You should have received a copy of the GNU General Public License
along with this program.  If not, see <https://www.gnu.org/licenses/>.
Modifications:
?>
<binary>
```

Does this work instead?  I did a trial run of just a few lines and it seemed to work well.

Reply:
Might have been too quick on the draw as you can see I had an update with this.

Reply:
I will let you handle this then. Not sure what you are doing to this code now.

Reply:
OK, I think this should work now.  I think there might be some weird things with the "<" and ">" around the gnu licenses link so I've taken them off.  I think this should be allowable since those are just noting the link.  Can you try this out?

Reply:
it worked thanks!